src/commands/case_management.py

```diff
@@ -0,0 +1,725 @@
+"""
+case_management.py - Case management commands for interaction on IRC.
+
+Copyright (c) 2018 The Fuel Rat Mischief,
+All rights reserved.
+
+Licensed under the BSD 3-Clause License.
+
+See LICENSE.md
+"""
+import datetime
+import functools
+import io
+import itertools
+import re
+import typing
+import uuid
+from datetime import timezone
+
+import humanfriendly
+from loguru import logger
+
+from ._list_flags import ListFlags
+from ..packages.commands import command
+from ..packages.context.context import Context
+from ..packages.epic import Epic
+from ..packages.permissions.permissions import (
+    require_permission,
+    RAT,
+    OVERSEER,
+    require_channel,
+)
+from ..packages.quotation.rat_quotation import Quotation
+from ..packages.rat import Rat
+from ..packages.rescue import Rescue
+from ..packages.utils import Platforms, Status
+
+_TIME_RE = re.compile(r"(\d+)[: ](\d+)")
```

Comment:
```suggestion
_TIME_RE = re.compile(r"(\d+)[: ](\d+)")
"""
Regex matcher used to find a time within a string. Used to determine
if a newly-submitted case is code red or not.
"""
```

Reply:
I was just working on answering this, but yes.  this is just a simple way to see if time was included in an inject, and then to auto-mark the case CR if included.